cms/app/assets/stylesheets/components/multi_combobox.css

```
@@ -0,0 +1,272 @@
+  min-height: 40px;
+  padding: 6px 10px;
+  border: 1px solid var(--mc-color-border);
+  border-radius: var(--mc-radius);
+  background-color: var(--mc-color-bg);
+  cursor: text;
+  transition: border-color 0.15s, box-shadow 0.15s;
+}
+
+.multi-combobox__input-area--focused {
+  border-color: var(--mc-color-border-focus);
+  box-shadow: 0 0 0 2px rgba(0, 119, 194, 0.2);
```

Stylelint (17.2.0)

[error] Expected "0.2" to be "20%" (alpha-value-notation)

(alpha-value-notation)